### PR TITLE
Filters for Sickrage

### PR DESCRIPTION
**Pull request by kicker83**
Added filters to remove year from title. The year was a problem to sickrage to snatch episodes.

### CODE REVIEW

File: src/Jackett.Common/Definitions/xbytes2.yml [hunk 1]
```diff
@@ -39,6 +39,8 @@
   search:
     path: index.php
     keywordsfilters:
+      - name: re_replace
+        args: ["S(\\d{1,2})E(\\d{1,2}) S(\\d{1,2})E(\\d{1,2})", "S$1/E$2"]
```
Comment (kaso17) [inline src/Jackett.Common/Definitions/xbytes2.yml]: This filter to workaround the sickrage bug is not acceptable. Please migrate to medusa or fix it either on the sickrage side or on the jackett side (for all indexers, not just one).

File: src/Jackett.Common/Definitions/xbytes2.yml [hunk 2]
```diff
@@ -57,6 +59,21 @@
             args: category
       title:
         selector: td[valign="middle"] a
+        filters:
+          - name: re_replace
+            args: ["S0(\\d{1,2})\\/E(\\d{1,2})", "$1x$2"]
```
Comment (kaso17) [inline src/Jackett.Common/Definitions/xbytes2.yml]: This probably will break other applications. Why are you rewriting S0XX/EYY to XXxYY?
Just stripping the / should be enough?!?

File: src/Jackett.Common/Definitions/xbytes2.yml [hunk 3]
```diff
@@ -57,6 +59,21 @@
...
+          - name: re_replace
+            args: ["\\(", ""]
+          - name: re_replace
+            args: ["\\)", ""]
+          - name: re_replace
+            args: ["2017", ""]
+          - name: re_replace
+            args: ["2018", ""]
+          - name: re_replace
+            args: ["2019", ""]
```
Comment (kaso17) [inline src/Jackett.Common/Definitions/xbytes2.yml]: I assume this is there in order to strip the year? sickrage can't handle years?
That sounds like a bad idea and will break other applications like sonarr/radarr.

**Comment by kicker83** [comment]
keep calm mate. I'll check your comments, and I'll try to do the yml as a
pro.

**Pull request closed without merging**